Utils/auto_update_docker/extract_affected_content_items.py

```diff
@@ -0,0 +1,447 @@
+            AND NOT pack.hidden
+            Return iss.docker_image, iss.path, iss.auto_update_docker_image, pack.path, pack.support
+            """
+        )
+    )
+
+
+def get_content_items_by_docker_image() -> dict[str, list[dict[str, Any]]]:
+    """Return all content items of type 'integration' and 'script', with their respective
+    docker images, support level, and pack path.
+
+    Returns:
+        dict[str, list[dict[str, Any]]]: The key will be the docker image, and the value will be a list
+        containing data about the content items and respective pack and support level.
+    """
+    content_images: dict[str, list[dict[str, Any]]] = defaultdict(list)
+    with ContentGraphInterface() as graph, graph.driver.session() as session:
+        content_items_info = session.execute_read(query_used_dockers_per_content_item)
+        for docker_image, content_item, auto_update_docker_image, full_pack_path, support_level in content_items_info:
+            if auto_update_docker_image:
+                content_item_py = Path(content_item).with_suffix(".py")
+                if content_item_py.is_file():
+                    # Since the full_pack_path is in the format "Packs/{pack path}"
+                    pack_path = full_pack_path.split("/")[1]
+
+                    # Since the docker image returned will include the tag, we only need the image
+                    docker_image_split = docker_image.split(":")
+                    docker_image_without_tag = docker_image_split[0]
+                    docker_image_tag = docker_image_split[1]
+                    content_images[docker_image_without_tag].append(
+                        {
+                            "content_item": content_item_py,
+                            "support_level": support_level,
+                            "pack_path": pack_path,
+                            "docker_image_tag": docker_image_tag,
+                        }
+                    )
+                else:
+                    logging.warning(f"{content_item_py} was returned from the graph, but not found in repo")
+            else:
+                logging.warning(f"{auto_update_docker_image=} configured for {content_item}, skipping")
+    return content_images
+
+
+def docker_tags_parser(benchmark_docker_tags: str) -> dict[str, str]:
+    if not benchmark_docker_tags:
+        return {}
+
+    # Remove spaces, if input has spaces between comma
+    # docker_tags_no_spaces = benchmark_docker_tags.replace(" ", "")
+
+    # Split the input string into key-value pairs
+    pairs = benchmark_docker_tags.split(",")
+
+    # Initialize an empty dictionary to store the key-value pairs
+    result_dict = {}
+
+    # Iterate over each key-value pair
+    for pair in pairs:
+        # Split the pair into key and value
+        key, value = pair.split(":")
+        # strip() to remove spaces
+        result_dict[key.strip()] = value.strip()
+
+    return result_dict
+
+
+@app.command()
```

Comment:
is there a way not to use a command here? Dont imaging we would add more commands to this script, we would just add more scripts